### PR TITLE
Overlaying cleaning

### PR DESCRIPTION
**Pull request by aicioara**
There was some leftover data in the `/quicktests/data` folder which was in fact used in the overlaying tests. Moved that in the overlaying folder, and aligned all `csv` files with this change.

### CODE REVIEW

File: quicktests/overlaying/tests/realistic/baseball.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ function makeData() {
 function run(svg, data, Plottable) {
   "use strict";
 
-  d3.csv("../overlaying/tests/baseball.csv").get(function(error, rows) {
+  d3.csv("/quicktests/overlaying/data/baseball.csv").get(function(error, rows) {
```
Comment (bluong) [inline quicktests/overlaying/tests/realistic/baseball.js]: Doesn't the prefixed forward slash point to the root directory?

File: quicktests/overlaying/tests/realistic/cities.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-  "use strict";  
+  "use strict";
 
 }
 
 function run(svg, data, Plottable) {
   "use strict";
 
-  d3.csv("../overlaying/tests/cities.csv").get(function(error, rows) {
+  d3.csv("/quicktests/overlaying/data/cities.csv").get(function(error, rows) {
```
Comment (bluong) [inline quicktests/overlaying/tests/realistic/cities.js]: As above

File: quicktests/overlaying/tests/realistic/hockey.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+  "use strict";
 
 }
 
 function run(svg, data, Plottable) {
   "use strict";
 
-  d3.csv("../overlaying/tests/hockey.csv").get(function(error, rows) {
-  var data = rows; 
+  d3.csv("/quicktests/overlaying/data/hockey.csv").get(function(error, rows) {
```
Comment (bluong) [inline quicktests/overlaying/tests/realistic/hockey.js]: As above

File: quicktests/overlaying/tests/realistic/stocks.js [hunk 4]
```diff
@@ -12,13 +12,13 @@ function run(svg, data, Plottable) {
   }
 
   // load GOOG
-  d3.csv("/quicktests/data/GOOG_20140401_20140901.csv")
+  d3.csv("/quicktests/overlaying/data/GOOG_20140401_20140901.csv")
     .get(function(error, rows) {
       var goog = rows.reverse();
       goog.forEach(processDatum);
 
       // load AAPL
```
Comment (bluong) [inline quicktests/overlaying/tests/realistic/stocks.js]: As above.  How did the previous test work...?

File: quicktests/overlaying/tests/realistic/baseball.js (hunk 1 above)
Comment (aicioara) [inline quicktests/overlaying/tests/realistic/baseball.js]: It does indeed. Which is fine. We basically changed from a relative path (bad) to an absolute one. We run grunt on the root folder anyway

File: quicktests/overlaying/tests/realistic/stocks.js (hunk 4 above)
Comment (aicioara) [inline quicktests/overlaying/tests/realistic/stocks.js]: The HTTP server we run will root to whichever folder we run grunt from. Hence, / is not the linux `/` but the server `/`

File: quicktests/overlaying/tests/realistic/baseball.js (hunk 1 above)
Comment (bluong) [inline quicktests/overlaying/tests/realistic/baseball.js]: Oh the root directory is considered to be the directory that `grunt` is run from?

Wait... that seems incorrect to me (What happens if I run `grunt` on a directory that contains Plottable)

**Comment by aicioara** [comment]
From the conversation:
> Relative paths are bad practice.
> consider what happens when you want to shift tests around
> create a different hierarchy 
> while leaving data in the same place
> you'll need to update all relative references

**Pull request merged**